.github/workflows/publish.yml

```
@@ -0,0 +1,11 @@
+name: Publish Artifacts
+
+on:
+  workflow_call:
+
+jobs:
+  publish:
+    if: startsWith(github.ref, 'refs/tags/')
+    permissions:
+      id-token: write
+    uses: dart-lang/setup-dart/.github/workflows/publish.yml@v1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)